planet/clients/orders.py

```diff
@@ -302,68 +302,51 @@ async def aggregated_order_stats(self) -> dict:
         resp = await self._do_request(req)
         return resp.json()
 
-    async def download_asset(self,
-                             location: str,
-                             filename: str = None,
-                             directory: str = None,
+    async def download_order(self,
+                             order: Order,
+                             directory: Path,
                              overwrite: bool = False,
-                             progress_bar: bool = True) -> str:
-        """Download ordered asset.
+                             callback=None,
+                             callback_args: tuple = None,
+                             callback_kwargs: dict = None) -> None:
```

Comment:
Like this we could support a kind of per-asset callback in general, not only a progress bar.